editor/plugins/tile_set_editor_plugin.cpp

```diff
@@ -569,12 +569,16 @@ void TileSetEditor::_on_textures_added(const PoolStringArray &p_paths) {
 	int invalid_count = 0;
 	for (int i = 0; i < p_paths.size(); i++) {
 		Ref<Texture> t = Ref<Texture>(ResourceLoader::load(p_paths[i]));
-		if (texture_map.has(t->get_rid())) {
-			invalid_count++;
+		if (t != NULL) {
+			if (texture_map.has(t->get_rid())) {
```

Comment:
That fix is fine, but I think it would look better not to add the null check and extra indentation, and to do instead:
```
Ref<Texture> t = Ref<Texture>(ResourceLoader::load(p_paths[i]));
if (!t.is_valid() || texture_map.has(t->get_rid())) {
    invalid_count++;
} else {
    ...
```
and maybe rename `t` to something slightly more explicit, like `tex`.